hostcontrol.js

```diff
@@ -7,16 +7,32 @@ const command = process.argv[2];
 
 (async function(command){
 	if (command == "--list"){
+		try{
 		console.log(await hosts.listAll())
 		process.exit(0)
+		} catch(e){
+			console.error(e)
+			process.exit(2)
```

Comment:
What does exit code `2` mean?

Reply:
Exit code 2 is for when the CLI does not complete its intended task. Prior to that the underlying hosts.js function should report what the error is. This has not been tested yet.

Reply:
After reviewing http://tldp.org/LDP/abs/html/exitcodes.html, i changed the exit code 2 to exit code 1. If the flag --help is used the script will exit with code 0, if theres a typo in the command the script will exit with code 127.